app/forms/submissions/trial/step03_content_form.rb

```diff
@@ -1,8 +1,9 @@
 module Submissions
   module Trial
     class Step03ContentForm < PlantTrialForm
-      # NOTE: needed because completely empty step breaks submission
-      property :_i_am_fake
+      property :lines_or_varieties, writeable: false
```

Comment:
That's right. I think in our case it does nothing as all properties are passed to `Submission#content` (otherwise they would be useless, wouldn't they?)